src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -236,7 +236,9 @@ static Command buildCommand(
     Collections.sort(outputFiles);
     Collections.sort(outputDirectories);
     command.addAllOutputFiles(outputFiles);
+    command.addAllOutputPaths(outputFiles);
     command.addAllOutputDirectories(outputDirectories);
+    command.addAllOutputPaths(outputDirectories);
```

Comment:
The `output_paths` must be sorted, see https://cs.opensource.google/bazel/bazel/+/master:third_party/remoteapis/build/bazel/remote/execution/v2/remote_execution.proto;l=596;drc=d0cba5507fcb5d636b1a9a3b1f58cf63314781c0.

Reply:
Good point! Fixed.